reTurn/AsyncTcpSocketBase.cxx

```diff
@@ -49,7 +49,7 @@ AsyncTcpSocketBase::connect(const std::string& address, unsigned short port)
    // into a list of endpoints.
    resip::Data service(port);
 #ifdef USE_IPV6
-   asio::ip::tcp::resolver::query query(address, service.c_str());   
+   asio::ip::tcp::resolver::query query(mSocket.local_endpoint().protocol(), address, service.c_str());
```

Comment:
is there a reason we can't just remove this #ifdef and always use mSocket.local_endpoint().protocol()?